Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -876,8 +879,12 @@ static bool isElementExcludedByRule(const MarkupExclusionRule& rule, const Eleme
     return matchedAttributes == rule.attributes.size();
 }
 
-bool MarkupAccumulator::shouldExcludeElement(const Element& element)
+bool MarkupAccumulator::shouldExcludeElement(const Element& element, Node& targetNode)
 {
+    if (targetNode.document().requiresScriptExecutionTelemetry(ScriptTelemetryCategory::FormControls)) {
```

Comment:
Why do we need targetNode? Surely, element.document() and targetNode.document() are the same document?

Reply:
That's true, I modified this line.